docker-compose.development.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.8"
+
+services:
+  todo-app:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: dev
+    image: todo-app:dev
+    env_file:
+      - .env.development

```

Comment:
Is there really a need for a separate .env.development file? .env should mainly be full of development environment settings (since it's what gets picked up when running `poetry run flask run` directly on your machine).

If you just want to tweak some things like FLASK_ENV between the two compose files, you can set that in the compose files.
```
    environment:
      - FLASK_ENV=development
    env_file:
      - .env
```
Values set in the compose file will take precedence over those in the env file.